### PR TITLE
IL-3455

### PR DESCRIPTION
**Pull request by mario-ipor**

### CODE REVIEW

File: contracts/vault/AssetManagementDai.sol [hunk 1]
```diff
@@ -8,6 +8,8 @@ contract AssetManagementDai is AssetManagement {
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
     uint256 public constant getVersion = 2_000;
+    uint256 private constant _SUPPORTED_NUMBER_OF_STRATEGIES = 3;
```
Comment (pete-ipor) [inline contracts/vault/AssetManagementDai.sol:11]: Use this in inside `_getNumberOfSupportedStrategies()`

File: contracts/vault/AssetManagementDai.sol [hunk 2]
```diff
@@ -8,6 +8,8 @@ contract AssetManagementDai is AssetManagement {
...
+    uint256 private constant _HIGHEST_APY_STRATEGY_ARRAY_INDEX = 2;
```
Comment (pete-ipor) [inline contracts/vault/AssetManagementDai.sol:12]: do you use this constant?

File: contracts/vault/AssetManagementDai.sol [hunk 3]
```diff
@@ -34,8 +34,12 @@ contract AssetManagementDai is AssetManagement {
         return 18;
     }
 
+    function _getNumberOfSupportedStrategies() internal view virtual override returns (uint256) {
+        return 3;
```
Comment (pete-ipor) [inline contracts/vault/AssetManagementDai.sol:38]: You add constant and don't use it?

File: contracts/vault/AssetManagementDai.sol (hunk 1 above)
Comment (pete-ipor) [reply contracts/vault/AssetManagementDai.sol:11]: maybe it should be public ?

**Review by pete-ipor — CHANGES_REQUESTED** [review]

**Review by pete-ipor — APPROVED** [review]

File: contracts/vault/AssetManagementDai.sol (hunk 1 above)
Comment (mario-ipor) [reply contracts/vault/AssetManagementDai.sol]: Redundant constant are removed

**Pull request merged**